src/s2/base/port.h

```diff
@@ -290,8 +290,8 @@ inline void sized_delete_array(void *ptr, size_t size) {
 
 #endif
 
-// defines __BYTE_ORDER for MSVC
-#ifdef _MSC_VER
+// defines __BYTE_ORDER for _WIN32
```

Comment:
There are a bunch of other places that use `_MSC_VER`:
https://github.com/google/s2geometry/search?q=_msc_ver

None of these need to change?

Reply:
Does `__MINGW32__` imply `_WIN32`?

Reply:
In my experience, yes.